src/Features/Customer/BalanceSheet/request.js

```diff
@@ -14,7 +14,7 @@ export const FetchBalances = () => {
   return useQuery(
     'balance',
     async () => {
-      const { data } = await axios.get(baseUrl() + `balance/user/${userId}`);
+      const { data } = await axios.get(baseUrl + `balance/user/${userId}`);
```

Comment:
we can use string interpolation instead of baseUrl + `` ---> `${baseUrl}balance/user/${userId}`;